Dockerfile

```diff
@@ -9,20 +9,29 @@
 #
 FROM ubuntu:20.04
 
+WORKDIR /pwndbg
+
+ENV LANG en_US.utf8
 ENV TZ=America/New_York
 RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && \
     echo $TZ > /etc/timezone && \
     apt-get update && \
-    apt-get -y install vim golang
-
-ADD . /pwndbg/
+    apt-get install -y locales && \
+    rm -rf /var/lib/apt/lists/* && \
+    localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8 && \
```

Comment:
When the Docker container is created and then run the test cases, pwngdb throws an alert about the UTF-8 is missing for Python. This fix it.